lib/apiservers/engine/backends/convert/stats.go

```diff
@@ -42,6 +43,11 @@ type ContainerStats struct {
 	curDockerStat *types.StatsJSON
 	currentMetric *metrics.VMMetrics
 
+	// disk & net stats are accumulated during the life of the
+	// subscription. These maps will assist in that accumulation.
+	diskStats map[string]metrics.VirtualDisk
+	netStats  map[string]metrics.Network
```

Comment:
Do we drop the ContainerStats when the container gets removed? Also how about restarts, do we just keep adding or somehow start from zero values?